database/controllers.js

```diff
@@ -0,0 +1,66 @@
+const { Client } = require('pg');
```

Comment:
Maybe you'll want to look into using Pool instead of Client. I believe Rafe mentioned something about Pool being able to handle more requests than Client, but feel free to do your own research

Reply:
Your're right. I looked it up and client has to do a handshake each time and pool doesn't. Refactored it to use pool